.github/workflows/node.js.yml

```diff
@@ -29,12 +29,6 @@ jobs:
       # Install dependencies
       - run: npm ci
```

Comment:
You no longer need to install dependencies here, as they're installed inside the container.